play-zipkin-tracing/play/src/main/scala/brave/play/TraceWSClient.scala

```diff
@@ -65,12 +65,22 @@ private class TraceWSRequest(spanName: String, request: WSRequest, tracer: Zipki
   override def withUrl(url: String): TraceWSRequest = new TraceWSRequest(spanName, request.withUrl(url), tracer, traceData)
   override def withMethod(method: String): TraceWSRequest = new TraceWSRequest(spanName, request.withMethod(method), tracer, traceData)
 
-  override def execute(): Future[Response] = tracer.traceFuture(spanName){ data =>
-    request.addHttpHeaders(tracer.toMap(data.span).toSeq: _*).execute()
-  }(traceData)
-  override def stream(): Future[Response] = tracer.traceFuture(spanName){ data =>
-    request.addHttpHeaders(tracer.toMap(data.span).toSeq: _*).stream()
-  }(traceData)
+  private def executeTraced(executeRequestFn: WSRequest => Future[Response]): Future[Response] = {
+    val tracingTags = List(
+      "http.method" -> this.method,
+      "http.path" -> this.uri.getPath,
+      "http.host" -> this.uri.getHost
```

Comment:
fyi this isn't usually tagged by default. remoteEndpoint is, though

Reply:
you mean we should call `remoteServiceName` in `ZipkinTraceServiceLike` ? Because right now it just calls `name` on the span and I see `remoteEndpoint` is deprecated.